drake/systems/controllers/QPController.m

```diff
@@ -346,13 +346,13 @@
 
       if include_angular_momentum
         A = centroidalMomentumMatrix(r, kinsol);
-        Adot_times_v = centroidalMomentumMatrixDotTimesVmex(r, kinsol);

```

Comment:
sorry you had to update these--this file's scheduled for deletion as soon as I finish porting all the Atlas examples to the new controller